MobiFlight/MidiBoard/MidiOutputDefinition.cs

```diff
@@ -34,7 +35,7 @@ public class MidiOutputDefinition
         /// <summary>
         /// The midi message ids. Possible value range from 0 to 127. Required.
         /// </summary>
-        public byte[] MessageIds;
+        public List<byte> MessageIds;
```

Comment:
This needs to be a `List` instead of a `byte[]` so when the object is converted to JSON for schema validation the `byte[]` doesn't get serialized as a `string`.